src/popup/components/NameItem.vue

```diff
@@ -319,7 +319,7 @@ export default defineComponent({
       }
 
       .truncate {
-        @extend %face-sans-14-bold;
+        @extend %face-sans-15-medium;
```

Comment:
Is this change coordinated with someone?

Reply:
with tsvetan